charts/istio/values.yaml

```diff
@@ -1,18 +1,28 @@
 # derived from https://github.com/knative/serving/blob/master/third_party/istio-1.3.3/values-lean.yaml
+# also see https://istio.io/docs/reference/config/installation-options/
 
 global:
   proxy:
     # Enable proxy to write access log to /dev/stdout.
     accessLogFile: "/dev/stdout"
     accessLogEncoding: 'JSON'
     autoInject: disabled
+    image: docker.io/istio/proxyv2:1.3.3
```

Comment:
rm, no need to redefine the default that will only make it harder to update to 1.3.4